dist/ci/travis_script.sh

```diff
@@ -47,14 +39,8 @@ if test -z "$SUBTEST"; then
       make test_unit
       ;;
     *)
-      make -C ../../ rubocop
-      bundle exec rails assets:precompile &> /dev/null
-      bundle exec rails test:api
-      bundle exec rails test:webui
-      bundle exec rspec
-      jshint .
-      unset DO_COVERAGE
-      bundle exec rails test:spider
```

Comment:
Do we want to get rid of this stuff? Doesnt this remove all the tests from travis?